scripts/upower/95-upower-hid.rules

```diff
@@ -150,6 +152,14 @@ ATTRS{idVendor}=="10af", ATTRS{idProduct}=="0001", ENV{UPOWER_BATTERY_TYPE}="ups
 ATTRS{idVendor}=="10af", ATTRS{idProduct}=="0004", ENV{UPOWER_BATTERY_TYPE}="ups"
 ATTRS{idVendor}=="10af", ATTRS{idProduct}=="0008", ENV{UPOWER_BATTERY_TYPE}="ups"
 
+# Arduino
+ATTRS{idVendor}=="2341", ATTRS{idProduct}=="0036", ENV{UPOWER_BATTERY_TYPE}="ups"
+ATTRS{idVendor}=="2341", ATTRS{idProduct}=="8036", ENV{UPOWER_BATTERY_TYPE}="ups"
+
+# Arduino
+ATTRS{idVendor}=="2A03", ATTRS{idProduct}=="0036", ENV{UPOWER_BATTERY_TYPE}="ups"
+ATTRS{idVendor}=="2A03", ATTRS{idProduct}=="8036", ENV{UPOWER_BATTERY_TYPE}="ups"
```

Comment:
It seems that defining more IDs in the sources got the upower rules file regenerated, but not all of this change was committed as part of the PR and so some of CI scenarios complained: see travis-ci.org/github/networkupstools/nut/jobs/774366109#L1994 for the expected changes.

IIRC, you need to re-run `autogen.sh` locally to have the file updated in the workspace.

Reply:
Posted that fix myself :)

FYI: It is generally recommended to use a separate branch (not same name as in an upstream project) for new features and sources of PRs, to reduce surprises in later codebase re-syncs :)